dataaccess/memory/schedule-access.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 The Gort Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package memory
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/getgort/gort/data"
+	"github.com/getgort/gort/telemetry"
+	"go.opentelemetry.io/otel"
+)
+
+//TODO(grebneerg): Actually assign unique IDs here
+
+func (da *InMemoryDataAccess) ScheduleCreate(ctx context.Context, command *data.ScheduledCommand) error {
+	tr := otel.GetTracerProvider().Tracer(telemetry.ServiceName)
+	_, sp := tr.Start(ctx, "memory.ScheduleCreate")
+	defer sp.End()
+
+	if command.ScheduleID != 0 {
+		return fmt.Errorf("Schedule id already set.")
```

Comment:
Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. 
See: https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#error-strings
This exists throughout, but I'll just add this comment here once.